model/wd2/index-respec.html

```diff
@@ -189,26 +189,32 @@
+  <dt><dfn data-lt="Specific Resource|Specific Resources">Specific Resource</dfn></dt>
+  <dd>As opposed to a <a>External Web Resource</a>, the state of a Specific Resource is entirely captured within the Annotation's representation.</dd>
 
-  <dt>Relationship</dt>
-  <dd>In the model sections, the term "relationship" is used to distinguish those features that refer to other Web Resources, either by reference to the resource's URI or by including a description of the resource in the Annotation's representation. The valid values for a Relationship are: a quoted string containing a URI, an object that has the "id" property, or an array containing either of these if more than one is allowed.</dd>
+  <dt><dfn data-lt="Property|Properties">Property</dfn></dt>
+  <dd>A feature of a <a>Web Resource</a>, that often has a particular data <a>type</a>. In the model sections, the term "property" is used to refer to only those features which are <em>not</em> <a>Relationships</a> and instead have a literal value such as a string, integer or date.  The valid values for a Property are thus any data type other than object, or an array containing members of that data type if more than one is allowed.</dd>
 
-  <dt>Class</dt>
-  <dd>A type that can be associated with a Resource. Classes are Web Resources, as they are identified by a URI.</dd>
+  <dt><dfn data-lt="Relationship|Relationships">Relationship</dfn></dt>
+  <dd>In the model sections, the term "relationship" is used to distinguish those features that refer to other <a>Web Resources</a>, either by reference to the resource's URI or by including a description of the resource in the Annotation's representation. The valid values for a Relationship are: a quoted string containing a URI, an object that has an "id" property, or an array containing either of these if more than one is allowed.</dd>
 
-  <dt>Instance</dt>
-  <dd>An instance of a particular Class. Instances are Resources, as they may or may not be identified by a URI.</dd>
+  <dt><dfn data-lt="Class|Classes">Class</dfn></dt>
+  <dd><a>Resources</a> may be divided, conceptually, into groups called "classes"; members of a class are known as <a>Instances</a> of the class. Resources are associated with a particular class through <a>typing</a>. Classes are identified by URI-s, i.e., they are also <a>Web Resources</a> themselves.</dd>
+
+  <dt><dfn data-lt="type|types|typing">Type</dfn></dt>
+  <dd>A special <a>Relationship</a> that associates an <a>Instance</a> of a class to the <a>Class</a> it belongs to.</dd>
+
+  <dt><dfn data-lt="Instance|Instances">Instance</dfn></dt>
+  <dd>An element of a group of <a>Resources</a> represented by a particular <a>Class</a>. Instances are identified by URI-s, i.e., they are also <a>Web Resources</a> themselves.</dd>
```

Comment:
Not to be too pedantic, but as this is the Terminology section ... a Blank Node could be an Instance, no? Thus Instances are Resources, not Web Resources.